pkg/controller/flink/client/api.go

```diff
@@ -36,7 +37,7 @@ const httpPost = "POST"
 const httpPatch = "PATCH"
 const retryCount = 3
 const httpGetTimeOut = 5 * time.Second
-const defaultTimeOut = 1 * time.Minute
+const defaultTimeOut = 5 * time.Minute
```

Comment:
is there a way to configure timeouts for different operations? I can see that for certain environments (minikube for instance) timeouts are prevalent.